src/adaptors/alpaca-finance/index.js

```diff
@@ -0,0 +1,71 @@
+const axios = require("axios");
+const utils = require('../utils');
+
+function formatSymbol(sourceName) {
+    const space = sourceName.indexOf(' ');
+    return`${sourceName.substring(space + 1)} (${sourceName.substring(0, space)})`;
+};
+
+async function apy(chain) {
+    const response = (await axios.get(
+        `https://alpaca-static-api.alpacafinance.org/${chain}/v1/landing/summary.json`
+        )).data.data;
+
+    const filteredStakingPools = response.fairLaunchStakingPools
+        .filter(p => !p.key.includes('debt'))
+    const fairLaunchStakingPools = filteredStakingPools.map(p => ({
+        pool: `${p.stakingToken.address}-staking`,
+        chain: chain == 'bsc' ? 'Binance' : utils.formatChain(chain),
+        project: 'alpaca-finance',
+        symbol: utils.formatSymbol(p.symbol),
+        tvlUsd: Number(p.tvl),
+        apy: Number(p.apy)
+    }));
+
+    const farmingPools = response.farmingPools.map(p => ({
+        pool: `${p.key}-farming-pool`,
+        chain: chain == 'bsc' ? 'Binance' : utils.formatChain(chain),
+        project: 'alpaca-finance',
+        symbol: formatSymbol(p.sourceName),
+        tvlUsd: Number(p.tvl),
+        apy: Number(p.totalApy / p.leverage)
+    }));
+
+    const ausdPools = response.ausdPools.map(p => ({
+        pool: `${p.key}-aUSD-pool`,
+        chain: chain == 'bsc' ? 'Binance' : utils.formatChain(chain),
+        project: 'alpaca-finance',
+        symbol: utils.formatSymbol(p.sourceName),
+        tvlUsd: Number(p.tvl),
+        apy: Number(p.totalApy)
+    }));
+
+    const lendingPools = response.lendingPools.map(p => ({
+        pool: `${p.ibToken.address}-lending`,
+        chain: chain == 'bsc' ? 'Binance' : utils.formatChain(chain),
+        project: 'alpaca-finance',
+        symbol: utils.formatSymbol(p.symbol),
+        tvlUsd: Number(p.tvl),
+        apy: Number(p.totalApy)
+    }));
+
```

Comment:
It would be nice if you could take the Automated Vaults into account. The data can be accessed via `response.strategyPools`. 👍👍👍

Reply:
ty for that, we will those as well :)

Reply:
> It would be nice if you could take the Automated Vaults into account. The data can be accessed via `response.strategyPools`. 👍👍👍

can you link to where these are on the UI? I couldnt find them, thanks

Reply:
<img width="1095" alt="Screenshot 2022-05-12 at 11 30 19" src="https://user-images.githubusercontent.com/74533329/168051123-c1595973-1845-4ddf-b722-047291e0c48d.png">

its up there, i couldn't find it either initially ^^